lib/logstash/inputs/http.rb

```diff
@@ -1,74 +1,73 @@
-# encoding: utf-8
-require "logstash/inputs/base"
-require "logstash/namespace"
-require "stud/interval"
-require "logstash-input-http_jars"
+require 'logstash/inputs/base'
+require 'logstash/namespace'
+require 'stud/interval'
+require 'logstash-input-http_jars'
 
-java_import "io.netty.handler.codec.http.HttpUtil"
+java_import 'io.netty.handler.codec.http.HttpUtil'
 
 # Using this input you can receive single or multiline events over http(s).
 # Applications can send a HTTP POST request with a body to the endpoint started by this
-# input and Logstash will convert it into an event for subsequent processing. Users 
+# input and Logstash will convert it into an event for subsequent processing. Users
 # can pass plain text, JSON, or any formatted data and use a corresponding codec with this
 # input. For Content-Type `application/json` the `json` codec is used, but for all other
 # data formats, `plain` codec is used.
 #
 # This input can also be used to receive webhook requests to integrate with other services
 # and applications. By taking advantage of the vast plugin ecosystem available in Logstash
 # you can trigger actionable events right from your application.
-# 
+#
 # ==== Security
 # This plugin supports standard HTTP basic authentication headers to identify the requester.
 # You can pass in an username, password combination while sending data to this input
 #
-# You can also setup SSL and send data securely over https, with an option of validating 
-# the client's certificate. Currently, the certificate setup is through 
-# https://docs.oracle.com/cd/E19509-01/820-3503/ggfen/index.html[Java Keystore 
+# You can also setup SSL and send data securely over https, with an option of validating
+# the client's certificate. Currently, the certificate setup is through
+# https://docs.oracle.com/cd/E19509-01/820-3503/ggfen/index.html[Java Keystore
 # format]
 #
 class LogStash::Inputs::Http < LogStash::Inputs::Base
-  require "logstash/inputs/http/tls"
+  require 'logstash/inputs/http/tls'
 
-  config_name "http"
+  config_name 'http'
 
   # Codec used to decode the incoming data.
   # This codec will be used as a fall-back if the content-type
-  # is not found in the "additional_codecs" hash
-  default :codec, "plain"
+  # is not found in the 'additional_codecs' hash
+  default :codec, 'plain'
 
   # The host or ip to bind
-  config :host, :validate => :string, :default => "0.0.0.0"
+  config :host, validate: :string, default: '0.0.0.0'
```

Comment:
we tend to stick to => notation for hashes in the logstash and logstash plugins codebase